activerecord/lib/active_record/persistence.rb

```diff
@@ -240,11 +240,15 @@ def upsert(attributes, returning: nil, unique_by: nil)
       #
       # ==== Examples
       #
-      #   # Insert multiple records, performing an upsert when records have duplicate ISBNs
+      #   # Given a Unique Index on books.isbn and the following record:
```

Comment:
Can also change books.isbn to `books.isbn`.

Reply:
here is not markdown but rdoc, it should be `<tt>books.isbn</tt>`.